visualization.spectrum/resources/language/resource.language.en_us/strings.po

```diff
@@ -95,3 +95,23 @@ msgstr "%i°"
 msgctxt "#30019"
 msgid "Turn continuously"
 msgstr "Turn continuously"
+
+msgctxt "#30020"
+msgid "Tilt"
+msgstr "Tilt"
+
+msgctxt "#30021"
+msgid "Base"
+msgstr "Base"
+
+msgctxt "#30022"
+msgid "%+i%%"
+msgstr "%+i%%"
+
+msgctxt "#30023"
+msgid "Field size"
+msgstr "Field size"
+
+msgctxt "#30024"
+msgid "%i%%"
+msgstr "%i%%"
```

Comment:
With formal labels was something changed inside Kodi and with this style now more supported (on Nexus) and need:
```po
msgid "{0:d}%"
```

Reply:
Done